napari/components/overlays/base.py

```diff
@@ -49,7 +51,9 @@ class CanvasOverlay(Overlay):
         The rendering order of the overlay: lower numbers get rendered first.
     """
 
-    position: CanvasPosition = CanvasPosition.BOTTOM_RIGHT
+    position: Union[
```

Comment:
This seems wrong... We should instead fix wherever things are not being coerced.

Reply:
This is because the brush circle overal uses a tuple for position:

https://github.com/napari/napari/blob/e0c0c8edd71b0f3b19c0df57e88c884d85ba968c/napari/components/overlays/brush_circle.py#L21

I'm not really sure what the right fix here is, perhaps someone with a bit more knowledge of overlays can help?

Reply:
Ah, I see... Yeah, that probably means we should move the position out of the base class :/

Ok, I think for the time being I would just merge the `Union` type as you suggested, this requires some thinking.